docs/Getting Started/Understanding Permissions/index.md

```diff
@@ -233,21 +298,35 @@ if (status[SDL.rpc.enums.FunctionID.GetVehicleData].getAllowedParameters()[SDL.r
 !@
 
 ### Observing Permissions
-If desired, you can set @![iOS]an observer!@ @![android,javaSE,javaEE,javascript]a listener!@ for a group of permissions. The @![iOS]observer's handler!@ @![android,javaSE,javaEE,javascript]listener!@ will be called when the permissions for the group changes. If you want to be notified when the permission status of any of RPCs in the group change, set the `groupType` to @![iOS]`SDLPermissionGroupTypeAny`!@ @![android,javaSE,javaEE]`PERMISSION_GROUP_TYPE_ANY`!@ @![javascript]`SDL.manager.permission.enums.PermissionGroupType.ANY`!@. If you only want to be notified when all of the RPCs in the group are allowed, set the `groupType` to @![iOS]`SDLPermissionGroupTypeAllAllowed`!@ @![android,javaSE,javaEE]`PERMISSION_GROUP_TYPE_ALL_ALLOWED`!@ @![javascript]`SDL.manager.permission.enums.PermissionGroupType.ALL_ALLOWED`!@.
+If desired, you can @![iOS]subscribe to!@@![android,javaSE,javaEE,javascript]set a listener for!@ a group of permissions. The @![iOS]subscription's handler!@ @![android,javaSE,javaEE,javascript]listener!@ will be called when the permissions for the group changes. If you want to be notified when the permission status of any of RPCs in the group change, set the `groupType` to @![iOS]`SDLPermissionGroupTypeAny`!@@![android,javaSE,javaEE]`PERMISSION_GROUP_TYPE_ANY`!@@![javascript]`SDL.manager.permission.enums.PermissionGroupType.ANY`!@. If you only want to be notified when all of the RPCs in the group are allowed, or go from allowed to some/all not allowed, set the `groupType` to @![iOS]`SDLPermissionGroupTypeAllAllowed`!@@![android,javaSE,javaEE]`PERMISSION_GROUP_TYPE_ALL_ALLOWED`!@@![javascript]`SDL.manager.permission.enums.PermissionGroupType.ALL_ALLOWED`!@.
 
 @![iOS]
 ##### Objective-C
 ```objc
-SDLPermissionObserverIdentifier observerId = [self.sdlManager.permissionManager addObserverForRPCs:@[<#RPC name#>, <#RPC name#>] groupType:<#SDLPermissionGroupType#> withHandler:^(NSDictionary<SDLPermissionRPCName, NSNumber<SDLBool> *> * _Nonnull change, SDLPermissionGroupStatus status) {
-    <#RPC group status changed#>
+SDLPermissionObserverIdentifier observerId = [self.sdlManager.permissionManager addObserverForRPCs:@[<#RPC name#>, <#RPC name#>] groupType:<#SDLPermissionGroupType#> withHandler:^(NSDictionary<SDLPermissionRPCName, NSNumber<SDLBool> *> * _Nonnull updatedPermissionStatuses, SDLPermissionGroupStatus updatedGroupStatus) {
```

Comment:
The Obj-C code snippet is still using `addObserverForRPCs:groupType:withHandler:` instead of `subscribeToRPCPermissions:groupType:withHandler:`

Reply:
Fixed